scripts/r.in.wms/wms_gdal_drv.py

```diff
@@ -35,9 +35,14 @@ def write(self, s):
 
 class WMSGdalDrv(WMSBase):
 
+    def __init__(self):
+        super().__init__()
```

Comment:
This is invalid syntax in Python 2 (still supported in master), see eg. https://www.pythonforbeginners.com/super/working-python-super-function

Reply:
@landam How do I do it, to make it suitable for Python2 and 3? Can I write:
```
try: super(WMSBase, self).__init__()
except: super().__init__()
```

Or better:
```
def __init__(self):
        self.temp_files_to_cleanup = []

        self.params = {}
        self.tile_size = {'bbox': None}

        self.temp_map = None
        self.temp_warpmap = None
        
        self.proxy = None
        self.proxy_user_pw = None
```
?

Reply:
Python2 syntax also works in Python3. Let's use Python2 syntax here. Later when we drop Python2 support it can be rewritten.

Reply:
Okay, I change it.

Reply:
Thank you Anika, neverthelesss, using `super()` in Python 2 requires that the parent class is a "new style class". If it isn't, a TypeError is being raised. You can easily test this by running the following under python 2:

``` python
class NewStyle:
    def __init__(self):
        super(NewStyle, self).__init__()
        print("OK")

class OldStyle:
    def __init__(self):
        super(OldStyle, self).__init__()
        print("OK")

instance_new = NewStyle()
instance_old = OldStyle()    # this raises!
```
To solve this, you need to also update `WMSBase`. This should do it:

``` diff
- class WMSBase:
+ class WMSBase(object):
```

You can read more about "new style" and "old style" classes here: https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python

Reply:
@pmav99 Thank you. Now Python 2 works for me.